Bottom: 03d40549d135f20b2cee99a28a5d5e224f845911
Top:    796c3b2a6da4dd6f5131a141e0c43ed33ef535fc
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-05-09 17:07:06 -0500

Add parameter for buildbot version



---
diff --git a/docker/buildbot-master/Dockerfile b/docker/buildbot-master/Dockerfile
index 521c823..dde9cdb 100644
--- a/docker/buildbot-master/Dockerfile
+++ b/docker/buildbot-master/Dockerfile
@@ -1,4 +1,5 @@
-FROM        buildbot/buildbot-master:v0.9.13
+ARG         BUILDBOT_VERSION=latest
+FROM        buildbot/buildbot-master:$BUILDBOT_VERSION
 MAINTAINER  Andrew Cooper <andrew.cooper@novatechweb.com>
 
 WORKDIR /buildbot
